devicetypes/smartthings/tint-remote-controller.src/tint-remote-controller.groovy

```diff
@@ -0,0 +1,310 @@
+ *
+ */
+
+metadata {
+	definition (name: "Tint Remote Controller", namespace: "smartthings", author: "SmartThings", mcdSync: true, ocfDeviceType: "x.com.st.d.remotecontroller") {
+
+		capability "Actuator"
+		capability "Color Control"
+		capability "Color Temperature"
+		capability "Configuration"
+		capability "Switch"
+		capability "Switch Level"
+		capability "Health Check"
+		capability "Light"
+
+		// Muller Licht
+        fingerprint inClusters: "0000, 0003, 1000", outClusters: "0003, 0004, 0006, 0008, 0300, 1000, 0019, 0000", manufacturer: "MLI",  model: "ZBT-Remote-ALL-RGBW", deviceJoinName: "Tint Remote Control"
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name:"switch", type: "lighting", width: 6, height: 4, canChangeIcon: true){
+			tileAttribute ("device.switch", key: "PRIMARY_CONTROL") {
+				attributeState "on", label:'${name}', action:"switch.off", icon:"st.lights.philips.hue-single", backgroundColor:"#00A0DC", nextState:"turningOff"
+				attributeState "off", label:'${name}', action:"switch.on", icon:"st.lights.philips.hue-single", backgroundColor:"#ffffff", nextState:"turningOn"
+				attributeState "turningOn", label:'${name}', action:"switch.off", icon:"st.lights.philips.hue-single", backgroundColor:"#00A0DC", nextState:"turningOff"
+				attributeState "turningOff", label:'${name}', action:"switch.on", icon:"st.lights.philips.hue-single", backgroundColor:"#ffffff", nextState:"turningOn"
+			}
+			tileAttribute ("device.level", key: "SLIDER_CONTROL") {
+				attributeState "level", action:"switch level.setLevel"
+			}
+			tileAttribute ("device.color", key: "COLOR_CONTROL") {
+				attributeState "color", action:"color control.setColor"
+			}
+		}
+		controlTile("colorTempSliderControl", "device.colorTemperature", "slider", width: 4, height: 2, inactiveLabel: false, range:"(2700..6500)") {
+			state "colorTemperature", action:"color temperature.setColorTemperature"
+		}
+
+		main(["switch"])
+		details(["switch", "colorTempSliderControl"])
+	}
+}
+
+private getGROUPS_CLUSTER() { 0x0004 }
+private getADD_GROUP_COMMAND() { 0x00 }
+private getMOVE_TO_COLOR_COMMAND() { 0x07 }
+private getON_OFF_ATTRIBUTE() { 0x0000 }
+private getMOVE_TO_COLOR_TEMPERATURE_COMMAND() { 0x0A }
+private getSTEP_COMMAND() { 0x02 }
+private getWRITE_ATTRIBUTES_COMMAND() { 0x02 }
+
+def parse(String description) {
+    def event = zigbee.getEvent(description)
+	def result = []
+    if (event) {
+        log.debug "Creating event: ${event}"
+       result = createEvent(event)
+    } else {
+        if (isAttrOrCmdMessage(description)) {
+            result = parseAttrCmdMessage(description)
+        } else {
+            log.warn "Unhandled message came in"
+        }
+    }
+	log.debug "Parse returned: ${result}"
+	return result
+}
+
+private parseAttrCmdMessage(description) {
+    def descMap = zigbee.parseDescriptionAsMap(description)
+    if (descMap.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER && descMap.attrInt == BATTERY_VOLTAGE_ATTRIBUTE) {
```

Comment:
This doesn't appear to be a battery device. So this and the battery parsing can be removed